cloudant-sync-datastore-android-encryption/src/main/java/com/cloudant/sync/sqlite/android/AndroidSQLCipherSQLite.java

```diff
@@ -47,7 +49,7 @@
      * @param provider Provider object that contains the key to encrypt the SQLCipher database
      * @return
      */
-    public static AndroidSQLCipherSQLite createAndroidSQLite(String path, KeyProvider provider) {
+    public static AndroidSQLCipherSQLite openOrCreate(File path, KeyProvider provider) {
```

Comment:
I agree they should be same, and I'm more inclined to go with `open` with the documented behaviour that it creates the file if it isn't there.